packages/model/addon/-private/model.js

```diff
@@ -372,14 +341,21 @@ const Model = EmberObject.extend(DeprecatedEvented, {
     @type {Boolean}
     @readOnly
   */
-  isValid: isValid,
+  @computed('currentState', 'errors.length')
```

Comment:
Once ff is on, we should remove `currentState` dependency

Reply:
I'll make sure to do that in the next pass (I'm going to make currentState a tracked property and all of these getters, it'll be a much simpler world while we wait for that flag to be on)